common/properties/coverageLevels.txt

```diff
@@ -155,6 +155,7 @@ tt ;	moderate ;	Tatar
 uk ;	modern ;	Ukrainian
 ur ;	modern ;	Urdu
 uz ;	modern ;	Uzbek
+uz_Cyrl ;	basic ;	Uzbek (Cyrillic)
```

Comment:
common/properties/coverageLevels.txt is automatically rebuilt, so shouldn't have manual changes.

Reply:
Er, it's in a manual commit but I ran the script to make the change. I'm used to re-generating artefacts in the commit that changes them rather than waiting for scheduled commit -- just to keep everything up to date. It's also a good validation check to make sure the change did the right thing.